test/spec/modules/spotxBidAdapter_spec.js

```diff
@@ -0,0 +1,379 @@
+import {expect} from 'chai';
+import adapterManager from 'src/adapterManager';
+import {spec, outstreamRender} from 'modules/spotxBidAdapter';
+import {parse as parseQuery} from 'querystring';
+import {newBidder} from 'src/adapters/bidderFactory';
+import {userSync} from 'src/userSync';
+import {config} from 'src/config';
+import * as utils from 'src/utils';
+import find from 'core-js/library/fn/array/find';
+
```

Comment:
Almost all of these imports are never used besides `expect` and `spec`

Can you remove the unused ones?


Both `CONSTANTS` as well as `INTEGRATION` below here are not used either.

Let me know if I missed something and there is a reason for keeping them!

Reply:
Updated. I went ahead and wrote a few more tests while I was in there to bump that coverage up a bit.

Reply:
Awesome!! Thanks for that.